tsrc/test/cli/test_sync.py

```diff
@@ -145,18 +146,17 @@ def test_copies_are_up_to_date(
     assert (workspace_path / "top.txt").read_text() == "v2"
 
 
-def test_copies_are_readonly(
+def test_copies_preserve_stat(
     tsrc_cli: CLI, git_server: GitServer, workspace_path: Path
 ) -> None:
     manifest_url = git_server.manifest_url
     git_server.add_repo("foo")
-    git_server.push_file("foo", "foo.txt", contents="v1")
+    git_server.push_file("foo", "foo.txt", contents="v1", mode=pygit2.GIT_FILEMODE_BLOB_EXECUTABLE)
```

Comment:
nit: just for clarity, consider naming the file `foo.exe` - then the assertion below will make more sense